src/organizerproxy.h

```diff
@@ -35,7 +35,8 @@ class OrganizerProxy : public MOBase::IOrganizer
   virtual MOBase::IModInterface *installMod(const QString &fileName, const QString &nameSuggestion = QString());
   virtual QString resolvePath(const QString &fileName) const;
   virtual QStringList listDirectories(const QString &directoryName) const;
-  virtual QStringList findFiles(const QString &path, const std::function<bool(const QString &)> &filter) const;
+  virtual QStringList findFiles(const QString &path, const std::function<bool(const QString &)> &filter) const override;
+  virtual QStringList findFiles(const QString& path, const QStringList& globFilters) const override;

```

Comment:
one & left and one & right, don't let isa see this

Reply:
See what? 🙄